src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
This is helpful because it lets me ensure that `MyStruct` implements `SIG`, but without making me ascribe it directly (which would destroy things that I need to be visible externally, such as datatype declarations).